index.html

```diff
@@ -26,7 +26,16 @@ <h1>Welcome to the student showcase!</h1>
         <div>SIX</div>
         <div>SEVEN</div>
         <div>EIGHT</div>
-        <div>NINE</div>
+        <div>NINE
+            <h2>Kelsey Sikes</h2>
+            <img src="https://scontent-ort2-1.xx.fbcdn.net/v/t1.0-9/10622820_300758826787698_7995727107980684309_n.jpg?_nc_cat=0&oh=3617b311860fea6fb3edecb566cb71a0&oe=5BF92FE5" alt="Kelsey">
+            <p>Favorite Quote: "The cure for bordeom is curiosity and there's no cure for that." - Richard P. Feynman</p>
+            <ul>
+                <li>
+                    <a href=mailto:ke_sikes@hotmail.com>Drop me an email!</a>
```

Comment:
Make sure that your `href` attribute has `"` around the `mailto:ke_sikes@hotmail.com`